sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/StorageAccountInfo.java

```diff
@@ -21,6 +22,20 @@ public class StorageAccountInfo {
     public StorageAccountInfo(final SkuName skuName, final AccountKind accountKind) {
         this.skuName = skuName;
         this.accountKind = accountKind;
+        this.isHnsEnabled = null;
+    }
+
+    /**
+     * Constructs a {@link StorageAccountInfo}.
+     *
+     * @param skuName SKU of the account.
+     * @param accountKind Type of the account.
+     * @param isHnsEnabled whether hierarchical namespace is enabled on the account.
+     */
+    public StorageAccountInfo(final SkuName skuName, final AccountKind accountKind, Boolean isHnsEnabled) {
```

Comment:
Do we want `isHnsEnabled` to be a `Boolean` or `boolean`, I know we've had some regrets with that in the past.

Reply:
Gone ahead with storing the value as boolean instead of Boolean.